git_repo/services/ext/gerrit.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
+from ..service import register_target, RepositoryService
+
+@register_target('gerrit', 'gerrit')
+class GerritService(RepositoryService):
+    fqdn = "localhost"
+    port = 29418
+    _max_nested_namespaces = None
+
+    def connect(self):
+        pass
+
+    @classmethod
+    def get_auth_token(cls, login, password, prompt=None):
+        return password
+
+    def get_repository(self, namespace, repo):
+        return namespace + '/' + repo
+
+    def is_repository_empty(self, project):
+        return False
```

Comment:
it is impossible to have an empty repo in gerrit?

Reply:
It is possible to have an empty repo gerrit. This part is not implemented yet and I will check if there is a way to find out if repository is empty via API.

Reply:
it's actually only used for clone, and to choose whether you want to start from scratch a project (`git init` in the project's directory) or actually pull from the remote. I'm not sure what's the process with gerrit, but I'm not sure it makes sense to create a new empty project on gerrit, does it?